src/server/src/main/java/io/cassandrareaper/service/SegmentRunner.java

```diff
@@ -791,6 +791,11 @@ private boolean handleJmxNotificationForCassandra22(
         break;
 
       case SUCCESS:
+
+        Preconditions.checkState(
+            !successOrFailedNotified.get(),
+            "multiple notifications of either 'SUCCESS' or 'FAILURE' are not valid");
+
```

Comment:
I would like to update (force push) this commit so that these lines contain the repairId and segmentId (so it's not so cryptic to debug in the event it does happen).
(The proposal is in https://github.com/thelastpickle/cassandra-reaper/commit/1cd25771dda94ffc7dfc908d215a4bd6b5df701d )

@adejanovski am i free to force push that commit to this branch?

Reply:
@michaelsembwever yes feel free to force push.

Reply:
thanks. done.